### PR TITLE
Faculty Image importer

### PR DESCRIPTION
**Pull request by jmbarne3**
Enhancements:
* Created a set of importers for pulling in profile images for faculty members from (1) WordPress exports of people custom post types and (2) a custom export format (CSV) from the College of Arts and Humanities' in-house personnel system.

### CODE REVIEW

File: importers/research-media-csv-importer.php [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * Imports thumbnails for researchers from WordPress Exports
```
Comment (cjg89) [inline importers/research-media-csv-importer.php:3]: Comment could be updated to note that this importer is for CSV exports

File: importers/research-media-csv-importer.php [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+		 * @param int $id The remote ID of the person
+		 * @return mixed
+		 */
+		private function get_person_image( $id ) {
+			$url = "{$this->media_base_url}/{$id}";
+			$media_data = $this->fetch_json( $url );
+
+			if ( $media_data ) {
+				$retval = $media_data->source_url;
+				if ( strpos( $retval, strval( $this->base_url ) ) == -1 ) {
```
Comment (cjg89) [inline importers/research-media-csv-importer.php:114]: Strict equality operator would be ideal here.  I'm also not sure that `strpos()` can ever return `-1`?

File: importers/research-media-csv-importer.php [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+				}
+
+				return $retval;
+			}
+
+			return false;
+		}
+
+		/**
+		 * Parses the WP Export File
```
Comment (cjg89) [inline importers/research-media-csv-importer.php:125]: Comment can be updated to specify CSV vs WP export file

File: importers/research-media-csv-importer.php [hunk 4]
```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * Imports thumbnails for researchers from WordPress Exports
+ */
+namespace UCF\MainSiteUtilities\Importers {
+	class CSVThumbnailImporter {
+		private
+			$csv_data,
+			$base_url,
+			$source_meta_key,
```
Comment (cjg89) [inline importers/research-media-csv-importer.php:10]: `$source_meta_key` doesn't appear to be in use anywhere in this class and could be removed

File: includes/commands.php [hunk 5]
```diff
@@ -46,5 +46,52 @@ public function import( $args, $assoc_args ) {
...
+		 *
+		 * [--force-update=<bool>]
+		 * : Determines if existing thumbnails will be deleted.
+		 *
+		 * [--use-cah-importer=<bool>]
+		 * : Whether or not to use the special CAH importer for their CSV file.
+		 *
+		 * ## EXAMPLES
+		 *
+		 *     wp research media-import wp-export.csv
```
Comment (cjg89) [inline includes/commands.php:72]: Example should be updated to use `thumbnails` command name (not `media-import`)

**Review by cjg89 — APPROVED** [review]

**Review by rjucf — APPROVED** [review]

**Review by cadie — APPROVED** [review]

**Pull request merged**